performance/font-loading.md

```diff
@@ -0,0 +1,67 @@
+# Font Loading
+
+## When to use webfonts
+
+Webfonts allow us to bring emotion, personality and tone to our sites - which can make them more successful.
+
+However, the task of choosing a webfont and implementing it on your site should be approached cautiously. Webfonts have the potential to impact aspects of your sites such as accessibility and page speed. For example:
+
+- Certain webfonts may reduce the readability of the site, for example for people with dyslexia, or low vision.
+- Webfonts can negatively impact page performance (additional http requests and increased page weight) especially in markets of strategic importance for Springer Nature like China. This will in turn impact usage numbers.
+
+Our recommendation is that you prioritise a [system font](https://css-tricks.com/snippets/css/system-font-stack/) over a webfont. System fonts are extensively tested, and incur no performance penalty.
+
+If a webfont is needed then close collaboration between user experience, accessibility, and web performance specialists must be sought to ensure the choice is satisfactory from typographic, accessibility, and performance perspectives.
+
+As a Frontend Developer, it's your role to provide technical guidance and feedback, not to choose the font.
+
+## Implementation
+
+We do not prescribe a particular implementation. Each team should determine which implementation is best suited for them. However, before selecting an implementation a team should be mindful of our working practices such as:
+
+- [Graded Browser Support](https://github.com/springernature/frontend-playbook/blob/main/practices/graded-browser-support.md)
+  - Is the implementation supported by all "enhanced" browsers? If not, is there a good justification for selecting it anyway?
+  - Does the implementation work in conjunction with the Cut The Mustard technique?
+- [Progressive Enhancement](https://github.com/springernature/frontend-playbook/blob/main/practices/progressive-enhancement.md)
+  - This is not meant to force you to choose a non-js implementation, but rather for you to ensure you consider what will happen for users in varying circumstances from the beginning.
+- [Our Performance Goals](https://github.com/springernature/frontend-playbook/blob/main/performance/performance-checklist.md)
+  - What impact will the implementation have on the page performance of your site? 
+  
+## Quick Tips
+Here are some quick suggestions for techniques that may prove useful:
+
+### Understand the different user experiences in font loading: FOIT, FOUT and FOIT
```

Comment:
Typo - second **FOIT** should be **FOFT**

Reply:
Good spot. 😂 
It's amazing how many times you can read something and not spot an obvious mistake like that.